File: Claude 3 Haiku/C++/2-2.c
#include <iostream>
#include <cmath>
using namespace std;

int main() {
    double radius, area;
    const double PI = 3.14159;

    cout << "Enter the radius of the circle: ";
    cin >> radius;

    area = PI * pow(radius, 2);

    cout << "The area of the circle with radius " << radius << " is " << area << " square units." << endl;

    return 0;
}

File: Claude 3 Haiku/python/2-2.py
import math

def calculate_circle_area():
    radius = float(input("Enter the radius of the circle: "))
    area = math.pi * radius ** 2
    print(f"The area of the circle with radius {radius} is {area:.2f} square units.")

if __name__ == "__main__":
    calculate_circle_area()
